packages/s3-presigned-post/src/createPresignedPost.integ.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { NoSuchKey, S3 } from "@aws-sdk/client-s3";
+const FormData = require("form-data");
+import { createReadStream, rmSync, writeFileSync } from "fs";
+import { join } from "path";
+
+import { createPresignedPost } from "./createPresignedPost";
+
+describe(createPresignedPost.name, () => {
+  const Bucket = "aws-sdk-js-integration-test-s3-presigned-post";
```

Comment:
The integration test is tied with account which owns the bucket.

Use bucket name from `AWS_SMOKE_TEST_BUCKET` instead, like it's done in existing e2e tests?